stablehlo/reference/Element.cpp

```diff
@@ -280,6 +280,21 @@ Element floor(const Element &el) {
   return Element(el.getType(), val);
 }
 
+Element imag(const Element &el) {
+  if (isSupportedFloatType(el.getType())) {
+    const llvm::fltSemantics &elSemantics = el.getFloatValue().getSemantics();
+    bool roundingErr;
```

Comment:
Put a note that this will be simplified once we have the utility to get an Element object out of a chosen value. #1154